content/docs/2.0/migration.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "Migration Guide"
++++
+
+## Migrating from KEDA v1 to v2
+
+Please note that you **can not** run both KEDA v1 and v2 on the same Kubernetes cluster. You need to [uninstall](../../1.5/deploy) KEDA v1 first, in order to [install](../deploy) and use KEDA v2.
+
+KEDA v2 is using a new namespace for it's Custom Resources: `keda.sh` instead of `keda.k8s.io` and introduces a new Custom Resource for scaling of Jobs. See full details on KEDA Custom Resources [here](../concepts/#custom-resources-crd).
+
+### Scaling of Deployments
+In order to scale `Deployments` with KEDA v2, you need to do only a few modifications to existing v1 `ScaledObjects` definitions, so they comply with v2:
+- change the value of `apiVersion` property from `keda.k8s.io/v1alpha1` to `keda.sh/v1alpha1`
+- rename property `spec.scaleTargetRef.deploymentName` to `spec.scaleTargetRef.name`
+
+Please see the examples below or refer to the full [v2 ScaledObject Specification](../concepts/scaling-deployments/#scaledobject-spec)
+
+ **Example of v1 ScaledObject**
+```yaml
+apiVersion: keda.k8s.io/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {scaled-object-name}
```

Comment:
Should we add the label here that was mandatory in the past and we are now removing (although not even using it anymore)?

Reply:
It is not mandatory for a few last v1 releases, so I am not sure whether we should mention it here?

Reply:
I'd say it can't hurt; we don't know what version folks are coming from?

Maybe add it but mention that this has not been required for a while and finally being removed?

Maybe overthinking here but we're not all always on latest version.

Reply:
Check the latest commit. Something like this?

Reply:
Perfect, thanks ❤️